api/controllers/auth/login.js

```diff
@@ -54,8 +54,10 @@ module.exports = function (req, res) {
          req.ab.log("successful auth/login");
          req.session.tenant_id = req.ab.tenantID;
          req.session.user_id = user.uuid;
+
+         // make response last so session is updated before next user interaction
+         await authLogger(req, "Local auth successful");
```

Comment:
Interesting. In theory, authLogger() isn't supposed to affect the session at all. But maybe writing to the log *after* sending the response was delaying api_sails from processing the next request in time?

There's a way to manually trigger saving the req.session data. I wonder if that could help here.
```js
req.session.save(() => {
    // Optional callback fn
    // ...
});
```
Normally, it does the save automatically with `res.send()` or `res.end()`. Yeah, I still don't fully understand what's causing this problem. But if this fixes it, that's great.